cmd/config.go

```diff
@@ -197,10 +172,37 @@ type Config struct {
 	SubscriberAgreementURL string
 }
 
+// ServiceConfig contains config items that are common to all our services, to
+// be embedded in other config structs.
+type ServiceConfig struct {
+	// DebugAddr is the address to run the /debug handlers on.
+	DebugAddr string
+	AMQP      *AMQPConfig
+}
+
+// AMQPConfig describes how to connect to AMQP, and how to speak to each of the
+// RPC services we offer via AMQP.
+type AMQPConfig struct {
+	Server            string
+	Insecure          bool
+	RA                *RPCServerConfig
```

Comment:
It seems kind of circular to have `c.VA.AMQP.VA`.  Is there a way we can flatten this out, or use names that make clear what the distinction is?